apps/dashboard/src/lib/wallet/nfts/types.ts

```diff
@@ -85,6 +86,50 @@ const moralisSupportedChainIdsMap: Record<number, string> = {
   [gnosis.id]: "",
   [gnosisChiadoTestnet.id]: "",
   [base.id]: "",
+  [polygonAmoy.id]: "",
```

Comment:
Since we are replacing these external services with our own indexer in the near future, I didn't want to add these chain objects to the SDK.

Also supported-chains should not be hard-coded, sadly Moralis and Alchemy do not provide an endpoint for such data.

Simplehash does, and there will be a PR for that.